### ISSUE TITLE
False positive format string warning

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#944]
```
(format "/blah/%s/blah?query=Luke%%20Skywalker" 1)
```

Workaround for now:

Turn the linter off with `{:linters {:format {:level :off}}}}` either locally in the namespace with `(ns foo {:clj-kondo/config '{...}})` or in `.clj-kondo/config.edn`.

**Comment by borkdude** [comment]
@cldwalker @mpenet I believe I have fixed the false positives with commit https://github.com/borkdude/clj-kondo/commit/91b6c55f5290fcfa9032ab221a6fc1b79af18827 on master. Could you please run this over your code base to check for any bad ones?

You can do so with:

```
$ clojure -Sdeps '{:deps {clj-kondo/clj-kondo {:git/url "https://github.com/borkdude/clj-kondo/" :sha "91b6c55f5290fcfa9032ab221a6fc1b79af18827"}}}' -m clj-kondo.main --lint src test
```

from the command line.

**Comment by cldwalker** [comment]
Works for us. Thanks!
